sourdough/resources/views/dashboardSide.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Queries and Reports Documentation</title>
+    </head>
+
+    <body>
+    <div style="width:500px;">
```

Comment:
just curious if this inline style was to show that it was an option or did it have some other purpose? I removed it locally and the webparts looked good. We usually don't encourage inline styling, so I'd suggest taking it out unless there was a specific usage for it that I wasn't noticing.

Reply:
Oops, an in-progress artifact. Thanks for the catch!